SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -472,3 +340,176 @@ void USpatialWorkerConnection::OnConnectionFailure()
 		GEngine->BroadcastNetworkFailure(GameInstance->GetWorld(), GetSpatialNetDriverChecked(), ENetworkFailure::FromDisconnectOpStatusCode(ConnectionStatusCode), *ErrorMessage);
 	}
 }
+
+bool USpatialWorkerConnection::Init()
+{
+	OpsUpdateInterval = 1.0f / GetDefault<USpatialGDKSettings>()->OpsUpdateRate;
+
+	return true;
+}
+
+uint32 USpatialWorkerConnection::Run()
+{
+	while (KeepRunning)
+	{
+		FPlatformProcess::Sleep(OpsUpdateInterval);
```

Comment:
Do we want to account for the time spent processing the ops?

Reply:
Hmmm, I don't think so since those calls are extremely quick. Do you think we should?

Reply:
Processing outgoing ops will involve reading/copying the entire contents of `OutgoingMessagesQueue`, so depending on how many things we're sending, it could be a lot of data.

Reply:
I think we can assume that is instantaneous. The bigger problem is the main thread and this going out of sync and eventually add an extra tick of latency. Could be improved by having the main thread cause this to happen and then have the interval as a starvation thing. Task for another time though.